homeassistant/components/plex/__init__.py

```diff
@@ -150,6 +151,7 @@ async def async_setup_entry(hass, entry):
     hass.data[PLEX_DOMAIN][DISPATCHERS].setdefault(server_id, [])
     hass.data[PLEX_DOMAIN][DISPATCHERS][server_id].append(unsub)
 
+    @callback
```

Comment:
This method is called by the Plex media server. This decorator doesn't do anything. 

I think that the current code is correct?

Reply:
I'm using the decorator more as a comment on the expected context.